eclipse/tests/src/nl/tudelft/watchdog/logic/interval/IntervalPersisterSingleIntervalTest.java

```diff
@@ -43,25 +37,11 @@ private void writeInterval() {
 		assertEquals(interval.getEnd(), savedInterval.getEnd());
 		assertEquals(interval.getDuration(), savedInterval.getDuration());
 		assertEquals(interval.isClosed(), savedInterval.isClosed());
-	}
 
-	private void compareIntervalAfterWrite() {
-		WatchDogItem savedItem = new ArrayList<>(persister.readItems())
-				.get(0);
-		assertTrue(savedItem instanceof IntervalBase);
+		assertEquals(savedInterval.getType(), savedInterval.getType());
```

Comment:
This assert can never fail.

Reply:
In addition, can we keep the method names as comments to understand what is going on?

Reply:
Ah, I copied the variables but changed the wrong name. Actually, the assertions were verifying the same thing multiple times. Therefore I removed the duplicate assertions and fixed the variables.